.envrc.sample

```diff
@@ -10,3 +10,6 @@ export CLUSTER_TOPOLOGY=true
 export EXP_RUNTIME_SDK=true
 export EXP_MACHINE_SET_PREFLIGHT_CHECKS=true
 export CLUSTER_NAME=test-dfkhje
+export SECRET_NAME=supersecret
```

Comment:
@michal-gubricky, thanks for this excellent work. I suggest targeting the `main` branch instead of `cspo`. Let's wait for #23, and then we can merge this one directly into the `main`. The reason is that PR#23 has received a thorough review, and merging additional code (unrelated to the review comments) in the middle of the review process might be confusing for the reviewers in #23.

Reply:
That sounds reasonable, I will incorporate changes from the review here and then create a new PR, which will target the `main` branch.

Reply:
After #23 was merged into `main`, also the target of this PR was changed automatically to targeting `main` branch